classes/module/Module.php

```diff
@@ -2881,6 +2901,8 @@ public function getTabs()
      * add a warning message to display at the top of the admin page.
      *
      * @param string $msg
+     *
+     * @return false|void
```

Comment:
Same here (to be confirmed) 

```suggestion
     * @return false|null
```